README.md

```diff
@@ -19,13 +19,21 @@ Install php dependencies using composer
 $ composer install
 ```
 
-Install project :
+Install project
 ```bash
 $ bin/console app:install
 $ yarn install && yarn build (or "yarn dev" for development usage)
 $ symfony server:start --no-tls
 ```
 
+Remove frontend (optional)
+
+When you don't need any frontend on your app with Symfony, this commands remove frontend part of Monofony.
```

Comment:
I would rephrase this to avoid any confusion with frontend = node part.

my take would be:

> Sometimes, your application does not need a public facing section. You just need the admin and everybody should be logged in before accessing that. This is what we call in Monofony „frontend”. 
> 
> When you don't need any frontend on your app with Symfony, this commands remove frontend part of Monofony.

Or rephrase entirely, without using frontend anymore. wdyt?

Reply:
> When building a SPA with Angular/React/VueJs or a mobile app, you may just need the administration panel and/or the API.
> This command basically leaves you only with the admin, protected routes and the API.
> The public part https://monofony.mobizel.com/ here is removed an redirected to the administration panel.

Reply:
this new version sounds much better.